examples/collection.json

```diff
@@ -72,15 +60,29 @@
     "datetime": {
       "minimum": "2015-06-23T00:00:00Z",
       "maximum": "2019-07-10T13:44:56Z"
-    }
+    },
+    "cf:parameter": [
+      {
+        "name": "sea_surface_temperature",
+        "unit": "K"
+      },
+      {
+        "name": "land_surface_temperature",
+        "unit": "K"
+      },
+      {
+        "name": "z",
```

Comment:
Shouldn't we put an example from the https://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html list?

Reply:
Yeah, that is a very valid point :-) Fixed!